qcodes/tests/dataset/test_dataset_basic.py

```diff
@@ -3,7 +3,7 @@
 import re
 from unittest.mock import patch
 import random
-from typing import Sequence, Dict, Tuple, Optional
+from typing import Sequence, Dict, Tuple, Optional, List
```

Comment:
is List used somewhere?
```suggestion
from typing import Sequence, Dict, Tuple, Optional
```

Reply:
Yes in test_dependent_parameters , no idea how that worked previously